### ISSUE TITLE
Downstream fix for installing Swatchbooker via Homebrew

### ISSUE THREAD

**Issue by luzpaz** [olivierberten/SwatchBooker#1]
From the Homebrew Swatchbooker formula: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/swatchbooker.rb#L31-L42

Would it be possible to add this to the next minor upgrade version ?

```
diff --git a/src/swatchbook/color.py b/src/swatchbook/color.py
index 48ae503..f3511a2 100644
--- a/src/swatchbook/color.py
+++ b/src/swatchbook/color.py
@@ -28,7 +28,7 @@ def dirpath(name):
    if not name:
        return name
    elif os.path.islink(name):
-       return os.path.dirname(os.path.abspath(os.path._resolve_link(name)))
+       return os.path.dirname(os.path.abspath(os.path.realpath(name)))
    else:
        return os.path.dirname(name)
```

**Comment by olivierberten** [comment]
Fixed in commit c3d49e5

**Comment by luzpaz** [comment]
Thanks @olivierberten! Quick question... if or when are you planning to do a minor update release? I'm asking because I'm gonna need to either way make a HEAD version of Swatchbooker Homebrew formula either way.
